test/kata6.joinNames.test.js

```diff
@@ -1,5 +1,17 @@
 const { joinNames } = require("../src");
 
 describe("joinNames", () => {
-  test("returns string of names, seperated by commas and an ampersand", () => {});
+  it("returns string of names, seperated by commas and an ampersand", () => {
```

Comment:
Would be good practice to add another test or 2 with different length arrays being passed in here